instantly/read.go

```diff
@@ -14,6 +14,10 @@ const (
 	objectNameAccounts  = "accounts"
 	objectNameEmails    = "emails"
 	objectNameTags      = "tags"
+
+	objectNameLeads            = "leads"
+	objectNameBlocklistEntries = "blocklist-entries"
+	objectNameUniboxReplies    = "unibox-replies"
```

Comment:
Since these constants are also used in delete and write, we should move to a new file that can be shared across actions.